foundry/https_download.py

```diff
@@ -1,3 +1,10 @@
+"""
+https_download.py
+
+Methods to download files from a Globus endpoint using Xtract (HTTPS). Now deprecated as of Aug 2022
```

Comment:
Do we need to say this replaces Xtract? I doubt it will help many people understand the code.

Reply:
Actually I think what this is saying is that this file is largely deprecated, because most of it uses xtract. I'd need to look closely at it to see if any of it is still used (I did not write this code, I just moved it out of Foundry a while back). 

I think this is a separate ticket.

Reply:
This does seem like a separate ticket. I'll resolve